shell/platform/embedder/embedder.h

```diff
@@ -2113,6 +2113,11 @@ typedef struct {
   /// and `update_semantics_callback2` may be provided; the others must be set
   /// to null.
   FlutterUpdateSemanticsCallback2 update_semantics_callback2;
+
+  /// If true, the Impeller rendering backend will be enabled. It is the callers
+  /// responsibility to verify that Impeller supports the currently selected
+  /// backend.
+  bool enable_impeller;
```

Comment:
TBD what sort of promise we want to make with this flag.

Reply:
That is how things already work.

Reply:
I missed this on initial review. I don't think we should add this flag because it will inevitably become outdated. Perhaps just allow it to be enabled through `command_line_argv`?

Reply:
I agree with @bdero. This is a flag that will only be valid in the interim during the transition. Piping it through the command line args makes more sense to me. We will also be able to flip the default at our convenience.

Reply:
Done